templates/wishlist.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>Wishlist</Name>
+  <Repository>wingysam/christmas-community</Repository>
+  <Registry>https://hub.docker.com/r/wingysam/christmas-community/</Registry>
+  <Network>bridge</Network>
+  <Shell>sh</Shell>
+  <Privileged>false</Privileged>
+  <Support>https://hub.docker.com/r/wingysam/christmas-community/</Support>
+  <Project>https://github.com/Wingysam/Christmas-Community/tree/master</Project>
+  <Overview>To create a simple place for your entire family to use to find gifts that people want, and to avoid double-gifting.&#xD;
+https://github.com/Wingysam/Christmas-Community/tree/master&#xD;
+Created by Wingysam&#xD;
+Converted by ajb3932</Overview>
```

Comment:
```suggestion
  <Overview>A web app for your family's Christmas shopping. Create a simple place for your entire family to use to find gifts that people want, and to avoid double-gifting.</Overview>
```